xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/objects/classes/ComputedFieldClass.java

```diff
@@ -90,6 +90,36 @@ public String getScript()
         return sValue;
     }
 
+    /**
+     * Computes and returns the raw value of this property for a given object.
+     * @param name property name
+     * @param prefix prefix to be added
+     * @param object object for which the property value has to get computed
+     * @param context current context
+     * @return the computed property value
+     * @since 11.8
+     */
+    public String getComputedValue(String name, String prefix, BaseCollection object, XWikiContext context) {
```

Comment:
Codestyle: `{` should be on a new line.

Reply:
Indeed, I fixed that and other formatting issues.